### PR TITLE
Renovated Npm - Make it possible to exclude modules from entrypoint

### PR DESCRIPTION
**Pull request by dxBeardedBear**

### CODE REVIEW

File: build/gulp/renovation-npm/index.js [hunk 1]
```diff
@@ -139,6 +139,7 @@ addCompilationTask({
     switches: {
         transpile: true
     },
+    components: 'all',
```
Comment (alexserov) [inline build/gulp/renovation-npm/index.js]: Consider `undefined` as default value
```suggestion
```

File: build/gulp/renovation-npm/index.js [hunk 2]
```diff
@@ -155,6 +156,7 @@ addCompilationTask({
     switches: {
         installPackages: true,
     },
+    components: ['Button'],
```
Comment (alexserov) [inline build/gulp/renovation-npm/index.js]: what if we use glob-style patterns ([minimatch](https://www.npmjs.com/package/minimatch)) here?

Comment (dxBeardedBear) [reply build/gulp/renovation-npm/index.js]: done

File: build/gulp/renovation-npm/index.js (hunk 1 above)
Comment (dxBeardedBear) [reply build/gulp/renovation-npm/index.js]: done

**Review by alexserov — APPROVED** [review]

**Pull request merged**